src/status_im/ui/screens/subs.cljs

```diff
@@ -41,19 +41,26 @@
 (reg-sub :peers-count :peers-count)
 (reg-sub :mailserver-status :mailserver-status)
 
+(reg-sub :fetching?
+         (fn [db]
+           (get db :inbox/fetching?)))
+
```

Comment:
Could use keyword as fn `(reg-sub :fetching? :inbox/fetching?)`

Reply:
I tried that before I think it doesn't work because of the namespace

Reply:
I think it's probably because you get the default value, as `reg-sub` fn has a arity of 2. You can only use the short version when a key is always granted to be there (non nilable)